router/options.go

```diff
@@ -1,37 +1,81 @@
 package router
 
-type Option func(r *chiWrapper)
+import (
+	"net/http"
 
-func OptEnableCORS(r *chiWrapper) {
-	r.enableCORS = true
+	"github.com/netlify/netlify-commons/tracing"
+)
+
+type Option struct {
+	f func(r *chiWrapper)
+}
```

Comment:
This is a change I'd actually keep, exposing the `Option` but not the `chiWrapper` is kinda..odd. And as it didn't work in the past b/c `chiWrapper` wasn't exposed, this makes it a safe breaking change (and will be doc'd that way).

Reply:
Mostly it is about the exposed signature. It is an exported type `Option` which has a signature using a non exported type `chiWrapper` which I think is messy. Functionally, not much changes as you can see